### PR TITLE
Add support for className attribute

### PR DESCRIPTION
**Pull request by czerasz**
Forward the `className` to the `<select>` tag

### CODE REVIEW

**Comment by rkit** [comment]
Maybe so?
```js
<select … {...this.props}>
``` 
Can will pass any properties such as id, data-* attributes

**Comment by lexich** [comment]
:+1:

**Comment by czerasz** [comment]
@rkit [Spread Attributes](https://facebook.github.io/react/docs/jsx-spread.html#spread-attributes) is a really good idea!

File: src/components/Select2.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class Select2 extends Component {
 
   render() {
     return (
-      <select multiple={this.props.multiple} defaultValue={this.props.defaultValue}>
+      <select {...props}>
```
Comment (rkit) [inline src/components/Select2.js]: Should be: `{..this.props}` but better maybe like that:

```js
render() {
  const { data, ...params } = this.props;
  return (
    <select {...params}>
      {data.map((item, k) => {
```
to exclude `data`

**Comment by rkit** [comment]
Thanks! Today will release.

**Comment by czerasz** [comment]
@rkit cool many thanks! :-)

**Pull request merged**